uktrade/data-hub-frontend
# Add notes on locations field

## Description of change

Add notesOnLocations field to location section of large capital profiles. This is 1 of 3 fields to be added to this section. 

## Test instructions
 
The 'Notes on investor's location preferences' field now appears after clicking edit on the Location section. When you save the text is saved and the paragraph breaks preserved. 

## Screenshots
### Before

![image](https://user-images.githubusercontent.com/42253716/62705667-c0837200-b9e5-11e9-9132-e40ab39bc69a.png)

### After 

![image](https://user-images.githubusercontent.com/42253716/62705544-81edb780-b9e5-11e9-842e-ba6aeb904d49.png)

![image](https://user-images.githubusercontent.com/42253716/62705597-9631b480-b9e5-11e9-8d8c-a9a7866164a6.png)

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied. 
https://github.com/uktrade/data-hub-frontend/blob/develop/docs/Code%20review%20guidelines.md"
